service-base/src/main/java/org/eclipse/hono/service/registration/BaseRegistrationService.java

```diff
@@ -139,9 +138,10 @@ public final void processRegistrationMessage(final Message<JsonObject> regMsg) {
             final JsonObject body = regMsg.body();
             final String tenantId = body.getString(MessageHelper.APP_PROPERTY_TENANT_ID);
             final String deviceId = body.getString(MessageHelper.APP_PROPERTY_DEVICE_ID);
-            final String action = body.getString(RegistrationConstants.FIELD_ACTION);
+            final String subject = body.getString(MessageHelper.SYS_PROPERTY_SUBJECT);
```

Comment:
this is the <em>operation</em>. we only use the `subject` property for conveying it ...